backends/transforms/remove_clone_ops.py

```diff
@@ -13,11 +13,24 @@
 
 def remove_clone_ops(graph: torch.fx.Graph) -> torch.fx.Graph:
     """
-    Remove clone op nodes and replace uses with parent node.
+    Remove clone op nodes that have the same dim_order as their input, and replace their uses with the input node.
     """
     clone_op = exir_ops.edge.aten.clone.default
+    clone_dim_order_op = exir_ops.edge.dim_order_ops._clone_dim_order.default
+
     for node in graph.nodes:
-        if node.op == "call_function" and node.target == clone_op:
+        if node.op != "call_function":
+            continue
+
+        # Identify clone_dim_order ops with unchanged memory layout.
```

Comment:
if we are supporting `aten.clone` elimination through this pass then we should similarly check `memory_format` arg.